packages/eslint-config-airbnb/README.md

```diff
@@ -40,6 +40,10 @@ If you use yarn, run `npm info "eslint-config-airbnb@latest" peerDependencies` t
   ```sh
   npm install -g install-peerdeps
   install-peerdeps --dev eslint-config-airbnb
+  
```

Comment:
let's close the highlighted section here, and start it anew below, since "or if" isn't valid shell syntax